gui-libs/wlroots/wlroots-9999.ebuild

```diff
@@ -60,20 +65,20 @@ BDEPEND="
 "
 
 src_configure() {
+	local backends="$(usex drm 'drm,' '')"
+	backends+="$(usex libinput 'libinput,' '')"
+	backends+="$(usex x11-backend 'x11,' '')"
+	# Get rid of a trailing comma
+	backends="${backends%,}"
 	# xcb-util-errors is not on Gentoo Repository (and upstream seems inactive?)
 	local emesonargs=(
-		"-Dxcb-errors=disabled"
 		-Dxcb-errors=$(usex xcb-errors enabled disabled)
 		$(meson_use tinywl examples)
 		-Drenderers=$(usex vulkan 'gles2,vulkan' gles2)
 		-Dxwayland=$(usex X enabled disabled)
-		-Dbackends=drm,libinput$(usex x11-backend ',x11' '')
+		-Dbackends="${backends}"
```

Comment:
You could, if you wanted, do this as an array instead and echo it w/ IFS here, but not required if you don't want to.

Reply:
Hmm, just've read about IFS and it seems to be exactly what we wanted. Will experiment a bit and try to implement it

Reply:
Decided to implement in the latest commit:
https://github.com/gentoo/gentoo/blob/ad83b87f3f988a2cf276e4a2659c241c48b0bfb1/gui-libs/wlroots/wlroots-0.16.2-r1.ebuild#L60-L66

Seems to work just fine. Thank you for the proposal, it looks much cleaner now!